src/js/bookmarkList.js

```diff
@@ -1,224 +1,219 @@
-var _ = require('underscore');
+let _ = require('lodash');
+let $ = require('jquery');
+
 require('ionic-framework');
 
 require('../../node_modules/ionic-framework/release/css/ionic.css');
 require('../vendor/animate.css/animate.css');
 require('../css/pocket.css');
 
-var Minilog = require('minilog');
-var common = require('./common');
-
-var log = Minilog('app');
-Minilog.enable();
+let bookmarksPaginator = require('./bookmarksPaginator');
+let bookmarksTransformer = bookmarksPaginator.BookmarksTransformer;
+let defaultBookmarksPaginator = bookmarksPaginator.UnreadCachedBookmarksPaginator;
+let currentBookmarksPaginator;
 
-var sort = 'newest';
-var state = 'unread';
+let common = require('./common');
 
-var CLEAN_CACHE_SEARCH_STRING = 'salsa:ccache';
 
 function isMobile() {
   return window.navigator.userAgent.indexOf('Mobile') !== -1;
 }
-window.angular.module('pocket', [
-  'ionic',
-])
 
-.controller('bookmarksCtrl', function($scope, $ionicLoading) {
+
+window.angular.module('pocket', [
+  'ionic'
+]).controller('bookmarksCtrl', function($scope, $ionicLoading) {
   if (!isMobile()) { // for debugging purposes
```

Comment:
forces the window open on chrome, so we can freely develop there without the need for kitt browser.